apps/developer-hub/src/components/BlankPage.tsx

```diff
@@ -0,0 +1,17 @@
+import { identity } from '@deso-core/identity';
+
+identity.configure({ identityURI: 'http://localhost:4201' });
```

Comment:
Most consuming apps will not need to call configure. They just import the instance and use it "as is" (now `new` calls needed, etc).